### DISCUSSION TITLE
BI with keystone

### DISCUSSION THREAD

**Discussion by arnold-almeida** [keystonejs/keystone#6602] [Event Q&A Q3 2021]
- Given {n} services with disparate schemas and views and layers
> mongo
> cassandra
> postgres, etc
> neo4j

Can we create an ETL pipeline (Snowflake) that surfaces the data as KeyStone manifold

-

**Comment by raveling** [comment]
@arnold-almeida we weren't 100% clear on the technical specifics of your question, but answered it as best we could at the recent community Q&A event:
https://www.youtube.com/watch?v=r1IJh-iHm1c&t=2925s
